README.md

```diff
@@ -1 +1,103 @@
-# openapi-msw
+<h1 align="center">OpenApi-MSW</h1>
```

Comment:
The stylized spelling is "OpenAPI", which could be used here. I think especially with the uppercase "OpenAPI-MSW" looks more pleasing and is more consistent in itself.